poller/index.js

```diff
@@ -41,6 +41,7 @@ const metricDefaults = {
   aligner: 'ALIGN_MAX',
   reducer: 'REDUCE_SUM'
 }
+const DEFAULT_THRESHOLD_MARGIN = 5;
```

Comment:
What about adding `regional_margin` and `multi_regional_margin` to `spannerDefaults`.  Those values will then automatically get merged into configuration, which should simplify things later on.

Reply:
I considered doing that, but it adds complexity to the default objects.
In the end, the margin will be added from the default constant in the same way as for custom metrics.

Reply:
Agreed it will largely end up the same, but I think it would be good to handle the default values consistently in each function.

Reply:
After discussing, I've add new issue (#28) to use the metrics defaults for the 3 default metrics.  We should revisit this code then.